docs/data/charts/tooltip/tooltip.md

```diff
@@ -69,6 +69,19 @@ Here is a demo with:
 
 {{"demo": "Formatting.js"}}
 
+### Axis formatter
+
+To modify how data is displayed in the axis use the `valueFormatter` property.
+
+Its second argument is a context which provides a `location` property with one of `'tick' | 'tooltip'` which you can use to customize the formatting.
```

Comment:
```suggestion
Its second argument is a context that provides a `location` property with either `'tick'` or `'tooltip'`.
```

- which -> that is grammarly suggestion
- `'tick' | 'tooltip'` is TS notation, better to avoid if not necessary since some devs did not yet moved to TS
- Not sure the end of the sentence is necessary

Those are personal opinion, feel free to pick or ignore them